### DISCUSSION TITLE
请问，QCefContext在哪里初始化的

### DISCUSSION THREAD

**Discussion by laoshiren0822** [CefView/QCefView#57] [Q&A] [answered]
作者你好，跑例子的时候崩溃了，源码中没有找到QCefContext初始化的位置。另外，Qt5.12.3 + vs2017 x86 编辑会有很多报错。比如说override以及std::lower，另外请问一下Linux可以通过吗？

**Comment by tishion** [comment, marked as answer]
1. 具体错误请贴出错误信息
2. https://github.com/CefView/QCefView/blob/main/src/QCefContext.cpp
3. Linux 可以编译通过，正常运行，但是有若干bug没有解决（X11相关），不适合用于正式产品发布
